entry-point.py

```diff
@@ -14,31 +15,32 @@ def dict_merge(dct, merge_dct):
     """
     for key, value in merge_dct.items():
         if (key in dct and isinstance(dct[key], dict)
-                and isinstance(merge_dct[key], collections.Mapping)):
+                and isinstance(merge_dct[key], collections.abc.Mapping)):
             dict_merge(dct[key], merge_dct[key])
         else:
             dct[key] = merge_dct[key]
 
 
-# Directories included into benchmarking suit
-directories = ["sample"]
-
-# JSON data, containing all the benchmarking results
-result = {}
-
-# Run benchmarks for each directory and collect everything into a single JSON
-for directory in directories:
-    os.chdir(directory)
-    subprocess.run(["bash", "-x", "run.sh"])
-    file = open("benchmark-results.json", "r")
-    dict_merge(json.loads(file.read()), result)
-    file.close()
-    os.chdir("-")
-
-# Write result into a file
-json_str = json.dumps(result)
-file = open("benchmark-results.json", "w+")
-file.write(json_str)
-file.close()
-# Fill Github Action output variable
-subprocess.run(["echo", f"::set-output name=report::benchmark-results.json"])
+if __name__ == '__main__':
+    # Directories included into benchmarking suit
+    directories = ["docker", "sample"]
+
+    # JSON data, containing all the benchmarking results
+    result = {}
+
+    # Run benchmarks for each directory and collect everything into a single JSON
+    for directory in directories:
+        os.chdir(directory)
+        subprocess.run(["bash", "-x", "run.sh"])
+        for bench_result in glob.glob("*benchmark-results.json"):
+            file = open(bench_result, "r")
+            dict_merge(result, json.loads(file.read()))
+            file.close()
+        os.chdir("..")
+
+    # Write result into a file
+    with open("benchmark-results.json", "w+") as file:
+        file.write(json.dumps(result, indent=4, sort_keys=True))
+        file.close()
+    # Fill Github Action output variable
+    subprocess.run(["echo", f"::set-output name=report::benchmark-results.json"])
```

Comment:
@Sammers21 you can just use `print()` to `stdout` instead of `echo` process

Reply:
@g4s8, Sounds logical, but Github Actions suggest to use echo:
![image](https://user-images.githubusercontent.com/16746106/90009610-0a610100-dca7-11ea-9d35-19e7558d0aef.png)

Reply:
Technically result is going to be the same, but I would prefer to follow Github documentation